changelog/6351.fixed.md

```diff
@@ -0,0 +1 @@
+Replace client.filter call in Jinja2 based computed attributes. Depending on schema and data this change will be yield a significant performance improvement for the computed attributes.
```

Comment:
I think the changelog should focus first on the user benefit more than the implementation, here it could be

```
Improve performance of computed attributes when updating a large number of objects at once. Replaced client.filter call in Jinja2 based computed attributes.
```

Reply:
Good call, I've updated the changelog. Will merge when the CI completes.